CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - insufficient_security_configurability.weak_password_policy.allows_password_to_be_same_as_email_username
 
 ### Changed
+- adjusted schema and added a (currently optional) CWE and CVSS base score for most entries
```

Comment:
Let's keep the changelog chronological, meaning appending changelog entries to the bottom of a particular section